src/views/List.jsx

```diff
@@ -68,15 +71,26 @@ export function List({ data, listToken, loading, logOut }) {
 		},
 	];
 
-	// get information by index
+	function handleCopy() {
+		setCopy(true);
+		navigator.clipboard.writeText(`${listToken}`);
+		toast.success('Copied!');
+		setTimeout(() => {
+			setCopy(false);
+		}, 2000);
+	}
 
 	return (
 		<div className="list-container">
+			<Toaster />
```

Comment:
I think you might be able to wrap Toaster around App in Index.jsx and then toast will be available on every component of the app.

Reply:
Good call!